### ISSUE TITLE
Easier way to find crash/orchestral cymbals from select instrument dialog

### ISSUE THREAD

**Issue by wizofaus** [musescore/MuseScore#23791]
### Your idea

It should be possible to find the instrument suitable for standard "crash cymbals" or "orchestral cymbals" by typing any of the names by which they're commonly known.

### Problem to be solved

I'm not sure what you're supposed to type though "hand cymbals" seems to map to the Muse Sounds "Piatti" sound which does indeed sound like crash cymbals.  
"Hand cymbals" to me suggests something much smaller and non-Western, e.g. https://www.exoticindiaart.com/product/sculptures/2-small-hand-cymbals-khartal-in-bronze-musical-instrument-miv971/?sitecurrency=USD&gad_source=1&gclid=CjwKCAjwko21BhAPEiwAwfaQCM-thYenQgNHhqJa0yse8DYeVB8bjJJwxsGbbQwldT_fXcu-Ck-9LhoCGzUQAvD_BwE 

### Checklist

- [X] This request follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting issues
- [X] I have verified that this feature request has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar requests

**Comment by bkunda** [comment]
It seems there's scope for more broadly improving search throughout the app, allowing the discovery of alternate item names. A more forgiving search query would make sense for something like this.

**Comment by wizofaus** [comment]
I do wonder about the use of "Hand cymbals" as the default term though - I've never heard them called that personally.

**Comment by zacjansheski** [comment]
Using "Hand Cymbal" is a concious choice to diffentiate the instrument from "Suspended Cymbal" or a drum kit's "Crash Cymbal"

https://en.wikipedia.org/wiki/Clash_cymbals

Clash Cymbals - This is pretty strictly European term. More and more, percussion instrument retailers are adopting the term "Hand Cymbals" https://www.steveweissmusic.com/category/hand-cymbals

There are several "correct" names for this instrument, so this is essentially a matter of opinion.

**Comment by wizofaus** [comment]
"cymbal pair" would be good.
I've never seen a score use "hand cymbal".

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by zacjansheski** [comment]
"Cymbal Pair" is not good.  (Hi-hats and finger cymbals are also "cymbal pairs")

**Comment by wizofaus** [comment]
I was thinking "orchestral crash cymbals (pair)".  The wiki talk page does seem to note that players generally don't use the term "clash cymbal".

**Comment by MarcSabatella** [comment]
I like use of the term “orchestral” - to me that could be sufficient even without the additional terms “crash” or “pair”.  But FWIW, Paiste calls them “hand cymbals”, Zildjian calls them “orchestral” as does Meinl “orchestral” (they use the term “symphonic more generically to also include suspended).

**Comment by wizofaus** [comment]
I agree the manufacturers seem to use that term but they're not really the typical MuseScore user!

**Comment by scoreorder** [comment]
I propose to use the instrument name that is common to use in orchestral scores. It is common to use the instrument name "Cymbals" in orchestral scores. It is not common to use the instrument name "Hand Cymbals" or the instrument name "Concert Cymbals" or the instrument name "Orchestral Cymbals" in orchestral scores. The major music publishers use the instrument name "Cymbals" in orchestral scores. I propose to use the instrument name "Cymbals".

See https://issuu.com/breitkopf/docs/pb_15140_issuu

**Comment by wizofaus** [comment]
In the concert of a score that's clearly orchestral music in the Western classical tradition that's obvious, sure. But notation software has to allow for a much broader scope.

**Comment by Valebjorg** [comment]
I know many scores that use "Cymbals", but not any scores that use "Hand Cymbals". A notation software should follow the common standard and use "Cymbals".

**Comment by MarcSabatella** [comment]
You mean, that's how you see them listed in *orchestral* scores.  In other contexts, the term "cymbals" implies other types of instruments than ioiorchestral hand cymbals, and hence the confusion that the diualoig is attempting *unncessfully) to avoid.

But as already explained above, "Cymbals" *is* how they appear in your score - there is no worry about the software not following the common standard.  When selecting that instrument for your orchestral scores, you get that instrument, labeled the conventional way.  No problem whatsoever.  The question is only about how they are listed *in the dialog* in a way that allows the context to be clear to all the different types of musicians out there writing music in different genres for different types of ensembles, not just for classical orchestra composers.

**Comment by Valebjorg** [comment]
A notation software should also work for orchestral musicians and composers.

The selection dialog should also follow the common standard and use "Cymbals". It is confusing for the user to select "Hand Cymbals" to get "Cymbals" in the score. The user should know which name to get in the score when making a selection. The name in the selection dialog and in the score should be the same. The user should select "Cymbals" to get "Cymbals" in the score.

**Comment by wizofaus** [comment]
It's not "also" if it ceases to work effectively for those working in other styles/genres (as I myself sometimes do).
I don't see how anyone can object to "Orchestral cymbals". And it's hardly the only instrument where the name for the "Change instrument" text needs to be changed (e.g. "To Marimba (single staff)").
